backend/src/routes/usersFriendsRoute.ts

```diff
@@ -122,6 +122,13 @@ friendshipRouter.get(
 
       await session.close();
 
+      if (users.length === 0) {
+        return res.status(404).json({
+          status: "not found",
+          message: "No users found with given queries",
```

Comment:
All other responses have a different format: `{ status: "error", errors: { "users": "not found" } }`.
In this case the page having no users isn't an error, the page is just empty

Reply:
Yes, you're right 😅
I tried to use different response codes, like 400 for bad requests and 404 for not found